### ISSUE TITLE
Reveal broken

### ISSUE THREAD

**Issue by drdevnull** [zurb/foundation-sites#7723]
The most up to date download breaks reveal...

1) download the complete package
2) copy the first example code from the reveal example page
3) paste it inside the index.html
4) run... see that it doesn't work

**Comment by drdevnull** [comment]
Update:
Modal can be displayed by hacking the JS:
$('#myModal').foundation('open')

I think the example on the site is wrong:
http://foundation.zurb.com/sites/docs/reveal.html

Also, the css is not hooked up properly either.

**Comment by drdevnull** [comment]
Note:
the examples here work:
http://foundation.zurb.com/sites/docs/reveal.html

I wrote it wrong, this example is broken:
http://foundation.zurb.com/sites/docs/v/5.5.3/components/reveal.html

**Comment by obadiola** [comment]
How can i configure the multiple open in Foundation 6?
this not working now
$(document).foundation({
        reveal: {
            multiple_opened: true
        }
    });

**Comment by zurbchris** [comment]
@obadiola 
```js
Foundation.Reveal.defaults.multipleOpened = true;
$(document).foundation();
```
@drdevnull what isn't working for you? Both docs pages (v6.1 & v5.5) seem to work just fine.

**Comment by jonathanmelville** [comment]
Noticing this as well. Reveal seems to be completely busted. 

I'm getting a strange behavior where it will trigger the overlay only but not the actual modal. Resizing the browser window causes the modal to appear.

Strangely if you trigger it with Javascript, it opens correctly: $('#exampleModal2').foundation('open')

**Comment by zurbchris** [comment]
@jonathanmelville @obadiola @drdevnull please checkout [this branch](https://github.com/zurb/foundation-sites/tree/update-reveal) and take a peak at the new `_setPosition` fn. More accurate, more performant, and ready to be battle tested before the 6.2 release.

**Comment by gakimball** [comment]
Fixed via 330018d577bfbdaf30dad96fa6a05e4a074419e6
